lms/static/js/student_account/views/account_settings_factory.js

```diff
@@ -29,7 +29,9 @@
                                 model: userAccountModel,
                                 title: gettext('Username'),
                                 valueAttribute: 'username',
-                                helpMessage: gettext('The name that identifies you on the edX site. You cannot change your username.')
+                                helpMessage: interpolate_text(
+                                    gettext('The name that identifies you on the {platform_name} site. You cannot change your username.'), {platform_name: platformName}
```

Comment:
@srpearce Do you think that there might be an issue with saying "the {platform_name} site" for all possible microsites? I'm thinking that depending on the platform's name, that sentence might not read very well.